bin/zowe-setup-certificates.sh

```diff
@@ -280,6 +280,20 @@ if ! [[ -z "${PKCS11_TOKEN_NAME}" ]] && ! [[ -z "${PKCS11_TOKEN_LABEL}" ]]; then
   fi
 fi
 
+echo "Creating GSKit keystore... STARTED"
+GSKIT_KEYSTORE="${KEYSTORE_PREFIX}.kdb"
+GSKKYMAN=gskkyman
+rm -f "${GSKIT_KEYSTORE}" "${KEYSTORE_PREFIX}.rdb" 2>/dev/null
+printf "1\n${GSKIT_KEYSTORE}\n${KEYSTORE_PASSWORD}\n${KEYSTORE_PASSWORD}\n\n\n0\n\n0\n0" | "${GSKKYMAN}" > /dev/null 2>&1
```

Comment:
these printfs need comments in case the structure of gskkyman ever changes.
what are the args that you are trying to apply?

looks like its 1 (create new db) <enter> name=keystore <enter> password=password <enter> reenter=password <enter> expiration=0 <enter> recordlength=default(5000) <enter> use fips mode=true/false (0=false) <enter>

Reply:
I added a comment.